packages/react-devtools-scheduling-profiler/src/content-views/NetworkMeasuresView.js

```diff
@@ -0,0 +1,337 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
+
+import type {NetworkMeasure, ReactProfilerData} from '../types';
+import type {
+  Interaction,
+  IntrinsicSize,
+  MouseMoveInteraction,
+  Rect,
+  ViewRefs,
+} from '../view-base';
+
+import {
+  durationToWidth,
+  positioningScaleFactor,
+  positionToTimestamp,
+  timestampToPosition,
+} from './utils/positioning';
+import {drawText} from './utils/text';
+import {formatDuration} from '../utils/formatting';
+import {
+  View,
+  Surface,
+  rectContainsPoint,
+  rectIntersectsRect,
+  intersectionOfRects,
+} from '../view-base';
+import {BORDER_SIZE, COLORS, SUSPENSE_EVENT_HEIGHT} from './constants';
+
+const HEIGHT = SUSPENSE_EVENT_HEIGHT; // TODO Constant name
+const ROW_WITH_BORDER_HEIGHT = HEIGHT + BORDER_SIZE;
+
+const BASE_URL_REGEX = /([^:]+:\/\/[^\/]+)/;
+
+export class NetworkMeasuresView extends View {
```

Comment:
would be interesting to learn more about this `View` abstraction at some point, i.e. learn how to use it / build new views, etc

Reply:
We can talk about it during the next standup if you'd like.